Skin/FrameXML/StaticPopup.lua

```diff
@@ -34,7 +34,7 @@ function private.FrameXML.StaticPopup()
 
         F.CreateBD(frame)
 
-        for j = 1, 3 do
+        for j = 1, 4 do
```

Comment:
I actually just started an update to the Static popup frames, so this won't be necessary.